spec/acceptance/firewall_attributes_exceptions_spec.rb

```diff
@@ -422,6 +423,22 @@ class {'::firewall': }
       end
     end
 
+    describe 'nflog_size' unless: iptables_version < '1.6.1' do
```

Comment:
Updating this line to
```
describe 'nflog_size', unless: iptables_version < '1.6.1' do
```
should resolve the syntax error :-)

Reply:
thank you so much for pointing this out, I was looking in all the wrong places for the error :)